### PR TITLE
 arrays lesson and practice

### PR DESCRIPTION
**Pull request by tccanini**

### CODE REVIEW

File: DemoArrayList/src/DemoList/OrangeArrays.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        sportEvents.add("Handball");
+        sportEvents.add("Ice Hockey");
+        sportEvents.add("Cricket");
+        sportEvents.add("Rugby");
+        sportEvents.add(3,"Soccer");
+
+        System.out.println("List of sport events after additions:" + sportEvents.size() );
+
+        // Display the array List
+        System.out.println("Contents of sportEvents:" + sportEvents);
```
Comment (edwinchukus) [inline DemoArrayList/src/DemoList/OrangeArrays.java]: You cannot print the container of ArraysList. Kindly review this line please.

File: DemoArrayList/src/DemoList/OrangeArrays.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+        //Remove elements from the array list
+        sportEvents.remove("Rugby");
+        sportEvents.remove(4);
+
+        System.out.println("List of sportEvents after deletion:" + sportEvents.size());
+        System.out.println("Content  of sportEvents:" + sportEvents);
```
Comment (edwinchukus) [inline DemoArrayList/src/DemoList/OrangeArrays.java]: You cannot print the container of ArraysList. Kindly review this line please.

**Review by edwinchukus — CHANGES_REQUESTED** [review]

**Comment by edwinchukus** [comment]
Kindly make these changes and revert back please.

**Comment by tccanini** [comment]
Ngozi ,

Please can you review this codes for me 

thank you

File: DemoArrayList/src/DemoList/OrangeArrays.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package DemoList;
+
+import java.util.ArrayList;
+
+/**
+ * Created by Kanini W'Tang on 7/18/2017.
+ */
+public class OrangeArrays {
+
+    public void ArrayListDemo (){
```
Comment (edwinchukus) [inline DemoArrayList/src/DemoList/OrangeArrays.java]: A method name must be an action word(s). and a method must start with lower case.

Comment (edwinchukus) [reply DemoArrayList/src/DemoList/OrangeArrays.java]: A method must do only one thing at a time.

**Review by edwinchukus — APPROVED** [review]
I am confident that the reviews and correction made and fine with me.

**Pull request merged**